bpo-43684: Add ADD_INT opcode (#25090)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2021-03-31-17-47-25.bpo-43684.hHmyyt.rst

Add an ``INT_ADD`` opcode. This combines ``LOAD_CONST`` for a small integer

> constant and ``BINARY_ADD`` into a single opcode, thus saving a modest

Reviewer: ```suggestion
constant and ``BINARY_ADD`` into a single opcode. Combining this popular pair of opcodes into one saves a modest amount of bytecode size and dispatch time.
```

> amount of bytecode size and dispatch time, given that this is a popular pair

Reviewer: ```suggestion
```

> of opcodes.

Reviewer: ```suggestion
```
